.github/workflows/check.yml

```diff
@@ -51,6 +50,18 @@ jobs:
         with:
           path: coverage-result/cobertura.xml
           minimum_coverage: 60
+      
+      - name: kairos-contracts x86_64-linux
+        if: matrix.os == 'ubuntu-latest'
+        run: nix build -L --no-link --show-trace .#packages.x86_64-linux.kairos-contracts
+
+      - name: kairos-contracts x86_64-darwin
+        if: matrix.os == 'macos-latest'
+        run: nix build -L --no-link --show-trace .#packages.x86_64-darwin.kairos-contracts
```

Comment:
Let's not add back x86_64-darwin.
aarch64-darwin is enough
```suggestion
```

Reply:
Will be addressed in other PR as discussed.